.changeset/slow-bears-fall.md

```diff
@@ -0,0 +1,8 @@
+---
+"go-web-app": patch
+---
+
+Fix comments in montandon landing page
```

Comment:
This is not relevant to user since they won't get the context of what "comment" is being talked about! Let's rewrite this whole changeset from the perspective of the general user!